src/cmd/src/cmdline.cpp

```diff
@@ -1508,4 +1524,4 @@ cmdline_parser_internal (
   
   cmdline_parser_release (&local_args_info);
   return (EXIT_FAILURE);
-}
+}
```

Comment:
The header of this file says it's automatically generated. I'm not sure if we should put custom code here. Why not put it on main.cpp?